tools/dockerfile/interoptest/grpc_interop_java_oracle8/build_interop.sh

```diff
@@ -16,16 +16,24 @@
 # Builds Java interop server and client in a base image.
 set -e
 
-mkdir -p /var/local/git
-git clone --recursive --depth 1 /var/local/jenkins/grpc-java /var/local/git/grpc-java
+cp -r /var/local/jenkins/grpc-java /tmp/grpc-java
 
 # copy service account keys if available
 cp -r /var/local/jenkins/service_account $HOME || true
 
-cd /var/local/git/grpc-java
-
+pushd /tmp/grpc-java
```

Comment:
copy from `/var/local/jenkins/grpc-java` directly into `/var/local/git/grpc-java/` ?

Reply:
No. I really do want to trash everything but the final binaries. I want the interop client to actually notice deployment issues.